### ISSUE TITLE
NEW PAGE : Night School : concept

### ISSUE THREAD

**Issue by sarmaswe** [eastbaydsa/website#166]
URLs: 
http://www.eastbaydsa.org/events/428/2018-07-12-socialist-night-school-housing-justice
http://www.eastbaydsa.org/resources/readings/socialist-school/2018/summer/session-1

We talked about creating a page for night school, though looking at how the night school events are on the calendar now, it seems to be pretty much working fine. 

What would it make sense to build here? A curriculum page that serves as an overview for use within our chapter and to point other chapter to for reference? How would that page then connect to the Events? Would we be interested in getting facilitators to film some night school content and then make it available, like an online class of sorts?

As is, it seems to be working, and night school is relatively new, so would just like to understand what we'd like to build, and then see how that lines up with other priorities. Also, if we start building pages for specific programs, how does that then apply to other programs? How do we determine which programs get their own pages? Does it make sense to build this page and not a page for our M4A work? Should we be looking at building something like a Programs page that would essentially offer different template for use across all programs?

<img width="1440" alt="screen shot 2018-07-12 at 9 13 16 am" src="https://user-images.githubusercontent.com/13740788/42645828-e66813b6-85b3-11e8-99f3-e9c0ccdff65e.png">

@dominic @wesholing @kyleschmolze @llkats

**Comment by sarmaswe** [comment]
One approach would be to set up a way to group/link events. So if we could tie all the night school events together, and then when someone clicked a Night School event, the Event page displayed the info about that event, and also context about the event group as well, with a list of related/past events. 

And then that same feature would apply to other events that occur as a series or repeat.

**Comment by sarmaswe** [comment]
@kyleschmolze are you thinking about it being something that's called out on the homepage, and then someone clicks it to go to Night-School specific page?

Similar to Single Payer on the homepage now?

http://www.eastbaydsa.org

**Comment by kyleschmolze** [comment]
Yeah something like that

**Comment by kyleschmolze** [comment]
But also, we'll want a process for keeping this stuff up to date, cause we def have 3 big campaigns right now and they're not on the homepage! I'm sure @wesholing has thought about this already and we can chat it out at the meeting next Wed

**Comment by wesholing** [comment]
Yup, I hounded co-chairs to put this page together, so they know about it
now. As things change, I'll update it.

As for the campaigns, I've asked about building them out and promoting
them, but apparently we need to be careful about how we promote things that
are PAC-related, like our electoral work, like Jovanka and Prop 10. This is
a good time to check in again, but that was the last I heard.

**Comment by sarmaswe** [comment]
@wesholing when you say you "hounded co-chairs to put this page together" -- what page are you talking about?

**Comment by kyleschmolze** [comment]
I think he was confusing threads - that looks like a comment about the Committees page, which he did have to hound the co-chairs for :)

**Comment by wesholing** [comment]
Oof, yes. I tried changing some settings in Gmail and ended up thoroughly confusing it (and me) when it came to these updates. That comment was related to the committees page, sorry about that. 

To touch on some points earlier in this thread:
- I really like @kyleschmolze's suggestion about a marketing page for Night School. We could even direct traffic to that from social media, email, etc.
- There's an effort being led by @sachaarnold right now (coordinating with me as a representative of the Political Education Committee) to reevaluate this section of the site. I think it might open up questions about our overall information architecture too, which we still need to address.

**Comment by sachaarnold** [comment]
I reviewed the website against the suggestions the outgoing Education Committee made.  See below for my own suggestions.

- I'd want to consult with the incoming EduCom folks to determine what categories to use for tagging the content.  For tagging the complexity, I saw "easy" and "moderate" used in 2017.  Is there interest in adding category and complexity tags for every reading?
- There appear to be only a handful of heading categories for each session page.  See below.  Ideas are welcome about what, if anything, might be added to the session page template implied by these.
-- Required Readings
-- Supplementary/Optional Readings
-- Historical Readings
-- Study Guide/Discussion Questions
-- Next Session
